ui/components/app/edit-gas-fee-button/index.scss

```diff
@@ -0,0 +1,48 @@
+.edit-gas-fee-button {
+  display: flex;
+  align-items: baseline;
+  justify-content: flex-end;
+
+  button {
+    @include H7;
+
+    display: flex;
+    align-items: baseline;
+    color: $primary-1;
+    background: transparent;
+    border: 0;
+    padding-inline-end: 0;
+    white-space: pre;
+  }
+
+  i {
+    color: $primary-1;
+    margin-right: 2px;
+  }
+
+  &__icon {
+    font-size: 1rem;
+  }
+
+  &__label {
+    font-size: 12px;
+    margin-right: 8px;
+  }
```

Comment:
[nit] maybe we should try to start staying consistent with either `rem` or `px` for these sizes? Was there a reason you used `1rem` for icon and `px` for everything else?

Reply:
I had a similar question but I wasn't sure about it. Do we have conventions around when to use `rem` vs. `px`?

Reply:
No specific reason to use `rem` for icon, as I remember it was some design feedback after which i changed the icon size to `1rem`.
I fixed the file to use `px` everywhere.

Reply:
Yeah would this `label` work using one of the `mixins`?

Reply:
I expect it to